playkit/src/main/java/com/kaltura/playkit/player/ImageTrack.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * ============================================================================
+ * Copyright (C) 2017 Kaltura Inc.
+ *
+ * Licensed under the AGPLv3 license, unless a different license for a
+ * particular library is specified in the applicable library path.
+ *
+ * You may obtain a copy of the License at
+ * https://www.gnu.org/licenses/agpl-3.0.html
+ * ============================================================================
+ */
+
+package com.kaltura.playkit.player;
+
+import androidx.annotation.Nullable;
+
+import com.kaltura.playkit.PKAudioCodec;
+
+/**
+ * Image track data holder.
+ *
+ */
+public class ImageTrack extends BaseTrack {
+    
+    private String label;
+    private long bitrate;
+    private int tilesHorizontal;
+    private int tilesVertical;
+    private float width;
+    private float height;
+    private long segmentDuration;
+    private long presentationTimeOffset;
+    private long timeScale;
+    private long startNumber;
+    private long endNumber;
+    private String imageTemplateUrl;
```

Comment:
@giladna @dan-ziv I see the model is different between web and android.
```
id: string,
    active: boolean,
    index: number,
    url: string,
    width: number,
    height: number,
    duration: number,
    rows: number,
    cols: number,
    customData: any
```
Can you please discuss and conclude what is the required model.
Preferably we should have one model and update accordingly for all platforms if needed.

@yairans FYI

Reply:
I refer to both naming inconsistencies and different attributes between models.

Reply:
I've removed all the dash-specific attributes from the model. They are all part of the `customData` container at run-time. Except this, the models are pretty much the same (different naming though).

Reply:
@OrenMe  any  reason to change the naming?